.eslintrc.changed.js

```diff
@@ -6,8 +6,8 @@ module.exports = {
     'jsx-a11y/anchor-is-valid': 2,
     'jsx-a11y/label-has-associated-control': 1,
     'jsx-a11y/no-static-element-interactions': 2,
-    '@department-of-veterans-affairs/prefer-telephone-component': 1,
-    '@department-of-veterans-affairs/telephone-contact-3-or-10-digits': 2,
+    '@department-of-veterans-affairs/prefer-telephone-component': 2,
+    '@department-of-veterans-affairs/telephone-contact-digits': 2,
```

Comment:
Have we updated the version of the ESLint plugin so that we get the changes?

Reply:
@bkjohnson I incremented to [v1.6.1](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/57de35a7e1f66cc1e29b55fd99ec7580dac04b27/packages/eslint-plugin/package.json#L3) did so I'm not sure why it's not picking it up. Published: https://www.npmjs.com/package/@department-of-veterans-affairs/eslint-plugin

Reply:
Maybe you mean in vets-website itself. I'll do that now but I thought it would pick up the minor bump.

Reply:
You'll probably need to do a `yarn install` to update the lock file.